config/crd/experimental/gateway.networking.k8s.io_backendtlspolicies.yaml

```diff
@@ -178,11 +178,11 @@ spec:
                 - hostname
                 type: object
                 x-kubernetes-validations:
-                - message: must not contain both CertRefs and WellKnownCACerts
-                  rule: (has(self.caCertRefs) && size(self.caCertRefs) > 0 && has(self.wellKnownCACerts)
-                    && self.wellKnownCACerts != "")
-                - message: must specify either CertRefs or WellKnownCACerts
-                  rule: '!(has(self.caCertRefs) && size(self.caCertRefs) > 0 || has(self.wellKnownCACerts)
+                - message: must not contain both CACertRefs and WellKnownCACerts
```

Comment:
I think we would want to change this at the source of truth: https://github.com/kubernetes-sigs/gateway-api/blob/adbd9968d4ff7995af21cb9cd0913288e2b4ca27/apis/v1alpha2/backendtlspolicy_types.go#L78-L79 ...followed by running ./hack/update-codegen.sh to generate the CRD yaml.